subprojects/parseq/build.gradle

```diff
@@ -65,9 +61,6 @@ sourceSets {
   test {
       compileClasspath += sourceSets.testUtils.output
       runtimeClasspath += sourceSets.testUtils.output
-      java {
-        srcDirs = ['src/test-utils/java']
```

Comment:
What is this change for?

Reply:
That's the test utility which should not be included into "test" source code. "test" and "test-utility" should be in separate directory. This is a previous configuration mistake.